pyomnisci/connection.py

```diff
@@ -1,22 +1,17 @@
 import omnisci
-import pandas as pd
-import pyarrow as pa
 
-from collections import namedtuple
 import base64
 import pandas as pd
 import pyarrow as pa
 import ctypes
-from sqlalchemy.engine.url import make_url
-from thrift.protocol import TBinaryProtocol, TJSONProtocol
-from thrift.transport import TSocket, TSSLSocket, THttpClient, TTransport
-from thrift.transport.TSocket import TTransportException
-from omnisci.thrift.OmniSci import Client, TCreateParams
```

Comment:
`Client` is referenced from tests/conftest.py, so we might need to keep it.

Reply:
Do we? Because in the test we have `mocker.patch("omnisci.connection.Client")` which refers to `omnisci` which is different than `pyomnisci`.

Reply:
I don't have the answer. Can you figure it out?

Reply:
Tests are all passing, resolving this.